ngrinder-runtime/src/main/java/org/ngrinder/http/HttpResponse.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2012-present NAVER Corp.
+ *
+ * This file is part of The nGrinder software distribution. Refer to
+ * the file LICENSE which is part of The nGrinder distribution for
+ * licensing details. The nGrinder distribution is available on the
+ * Internet at https://naver.github.io/ngrinder
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ngrinder.http;
+
+import org.apache.hc.client5.http.async.methods.SimpleHttpResponse;
+import org.apache.hc.core5.http.ProtocolVersion;
+import org.ngrinder.http.util.JsonUtils;
+
+import java.util.Map;
+
+public class HttpResponse {
+	private final SimpleHttpResponse response;
+
+	HttpResponse(SimpleHttpResponse response) {
+		this.response = response;
+	}
+
+	public byte[] getBodyBytes() {
+		return response.getBodyBytes();
+	}
+
+	public Map<?, ?> getBodyJson() {
+		return JsonUtils.deserialize(getBodyBytes());
+	}
```

Comment:
Do we really need body json code in HttpResponse level?
What about provide some mechanism for user to get json in their favorite way?

such as..

```
   response.getBody(asJson());
```
We can provide ```asJson ``` using groovy json library.
If we do this, we can just take out the jackson dependency in runtime.

Reply:
Groovy and Jython both supports json parser as built-in library. They provide a convert method `String` to `Map`(=json).

If we provide method gets the converter as a parameter like below, we can support json as a way integrated with each script engine.

**HttpResponse**
```java
public <T> T getBody(Function<String, T> converter) {
	return converter.apply(getBodyText());
}
```

**Groovy**
```groovy
import java.util.function.Function

import groovy.json.JsonSlurper

def jsonSlurper = new JsonSlurper()
def asJson = new Function<String, Map<?, ?>>() {
	public Map<?, ?> apply(String s) {
		return jsonSlurper.parseText(s)
	}
};

...
def json = httpResponse.getBody(asJson())
```

**Jython**
```python
from java.util.function import Function

import json

class JsonConverter (Function):
	def apply(self, s):
		return json.loads(s)

asJson = JsonConverter()

...
json = httpResponse.getBody(asJson())
```

It seems simple and sufficient to meet user requirements. I'm gonna remove Jackson dependency in `ngrinder-runtime`.